cocos/editor-support/spine-creator-support/SkeletonRenderer.cpp

```diff
@@ -150,16 +150,23 @@ SkeletonRenderer::~SkeletonRenderer () {
 }
 
 void SkeletonRenderer::destroy() {
+    stopSchedule();
     CC_SAFE_RELEASE_NULL(_effectDelegate);
     if (_ownsSkeletonData) {
-        delete _skeleton->getData();
+        if (_skeleton) {
+            delete _skeleton->getData();
+        }
         _ownsSkeletonData = false;
     }
     if (_ownsSkeleton) {
         CC_SAFE_DELETE(_skeleton);
+    } else {
+        _skeleton = nullptr;
     }
     if (_ownsAtlas && _atlas) {
         CC_SAFE_DELETE(_atlas);
+    } else {
+        _atlas = nullptr;
     }
```

Comment:
May be it is better to modify all codes like this
```c++
if (_ownsSkeleton) {
    delete(_skeleton);
}
if (_ownsAtlas) {
    delete _atlas;
}
...

_skeleton = nullptr;
_atlas = nullptr;
...
```

- no matter need to delete the instance, the pointer should be set to null.
- delete a null pointer is not a problem.